.drone.yml

```
@@ -928,13 +928,10 @@ trigger:
 
 steps:
   - name: build-docs
-    image: plugins/hugo:latest
-    pull: always
+    image: golang:1.19
     commands:
-      # https://github.com/drone-plugins/drone-hugo/issues/36
-      - apk upgrade --no-cache libcurl && apk add --no-cache make bash curl
       - cd docs
-      - make trans-copy clean build
+      - cd docs && make trans-copy clean build
```

```suggestion
      - make trans-copy clean build
```
the step above cd's into docs

---

Ah I missed that. Should probably merge these targets into the main Makfile so cd'ing around becomes unneccesary, but I'm deferring that for later.